### ISSUE TITLE
Allow multiple samples/variants

### ISSUE THREAD

**Issue by bennyn** [welovecoding/vote4code#55]
For some fights it could happen that there are multiple variantes to demonstracte variant A & variant B, thus vode4code should allow to add even more snippets to the same fight.

**Comment by lipis** [comment]
nah.. it will complicate stuff for no reason :) It's `1 vs 1` :)

**Comment by bennyn** [comment]
Ok, let's leave this for now. :D

But we should consider samples with consits of multiple files by default.
